.github/workflows/docker.yaml

```diff
@@ -55,6 +55,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: mrc-ide/naomi
+          ref: metadata-without-data
```

Comment:
Revert this once Naomi is merged. (Or maybe comment it out to make it easier to find next time)